### PR TITLE
Fix report_data API with empty AWS provider

### PR DESCRIPTION
**Pull request by adberglund**
This is a port of https://github.com/project-koku/masu/pull/479 into koku. This just allows report_data API run against all providers to not crash on an AWS provider that doesn't actually have data.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -86,9 +86,8 @@ deps =
   codecov
 commands =
   /bin/sh {toxinidir}/scripts/create_test_db_user.sh {env:DATABASE_USER} {env:DATABASE_ADMIN}
-  pipenv run pip install pip==18.0
   pipenv install --dev  --ignore-pipfile
-  coverage run --parallel-mode --concurrency=multiprocessing {toxinidir}/koku/manage.py test --noinput --parallel -v 2 {posargs: koku/masu/}
+  coverage run --parallel-mode --concurrency=multiprocessing {toxinidir}/koku/manage.py test --noinput --parallel -v 2 {posargs: masu.test.database.test_aws_report_db_accessor.ReportDBAccessorTest.test_populate_line_item_daily_summary_table_no_bill_ids}
```
Comment (dccurtis) [inline tox.ini]: Forgot to undo this file

Comment (adberglund) [reply tox.ini]: I specifically staged a highlighted area to make sure this DIDN'T happen, and it still did. <ugh>

Comment (adberglund) [inline tox.ini]:
```suggestion
  coverage run --parallel-mode --concurrency=multiprocessing {toxinidir}/koku/manage.py test --noinput --parallel -v 2 {posargs: koku/masu/}

```

File: tox.ini [hunk 2]
```diff
@@ -86,9 +86,9 @@ deps =
   codecov
 commands =
   /bin/sh {toxinidir}/scripts/create_test_db_user.sh {env:DATABASE_USER} {env:DATABASE_ADMIN}
-  pipenv run pip install pip==18.0
   pipenv install --dev  --ignore-pipfile
   coverage run --parallel-mode --concurrency=multiprocessing {toxinidir}/koku/manage.py test --noinput --parallel -v 2 {posargs: koku/masu/}
+
```
Comment (adberglund) [inline tox.ini]:
```suggestion
```

**Pull request merged**